main.js

```diff
@@ -99,6 +99,8 @@ ODate.prototype.update = function() {
 		dateString = ODate.format(date, 'date');
 	} else if (format === 'time-ago-limit-4-hours'){
 		dateString = ODate.timeAgo(date, { limit: 4*inSeconds.hour });
+	} else if (format !== '') {
```

Comment:
This conditional would mean we fall into this branch of code if `data-o-date-format` is not set, which is what the else statement is meant to catch.

If `data-o-date-format` is not set, it returns `null`, perhaps this conditional should check if `format !== null`, meaning it has been set to something.